src/context/AvailabilityContext.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState, createContext } from 'react'
+import Maintenance from '../components/Maintenance'
+
+const AvailabilityContext = createContext()
```

Comment:
COMMENT: maybe add a reference to availability of backend? AvailabilityContext seams too generic, no?